test_scripts/API/SetMediaClockTimer/ATF_P_SetMediaClockTimer_enableSeek_False.lua

```diff
@@ -0,0 +1,131 @@
+--------------------------------------------------------------------------------------------
```

Comment:
@OVikhrov, script crashes on start with:
Lua error:
./user_modules/shared_testcases/commonFunctions.lua:330: attempt to perform arithmetic on global 'NewTestSuiteNumber' (a nil value)